### PR TITLE
Added security requirements.

### LINKED ISSUE #69: Task - SRS - 15 Security
**Description**
- station access, data privacy
**Definition of Done**

### PR DESCRIPTION
**Pull request by RafeedIqbal**
Add some security requirements. lmk if anything needs to be added, removed or changed.

closes #69

### CODE REVIEW

File: docs/SRS-Volere/SRS.tex [hunk 1]
```diff
@@ -744,15 +744,42 @@ \subsection{Adaptability Requirements}
...
 \subsection{Audit Requirements}
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item The application will adhere to any and all audit rules followed by City 
+  departments.
+\end{enumerate}
 \subsection{Immunity Requirements}
-\lips
+\begin{enumerate} [{SR-IR}1.]
+  \item The application will be made as secure as feasibly possible within the
```
Comment (Spitgranger) [inline docs/SRS-Volere/SRS.tex:779]: Is this actually verifiable? Maybe mention some standards or saying that all components of the system should be automatically patched if there is a vulnerability found

Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex]: Ok, then could you or @kyle4908 replace it with "Security vulnerabilities and weaknesses should be patched as soon they are identified." and make the commit, I don't have the formatter installed and it fails the linter.

Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex]: nvm kyle helped me install the formatter.

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -744,15 +744,41 @@ \subsection{Adaptability Requirements}
...
-\lips
+\begin{enumerate} [{SR-AR}1.]
+  \item The application will adhere to any and all audit rules followed by City
+    departments.
+\end{enumerate}
 \subsection{Immunity Requirements}
-\lips
+\begin{enumerate} [{SR-IMR}1.]
+  \item Security vulnerabilities and weaknesses should be patched as
+    soon they are identified.
```
Comment (kyle4908) [inline docs/SRS-Volere/SRS.tex:780]: nit: we should maybe include a time frame IMO. Like expected to patch within a week or something like that.

**Review by kyle4908 — APPROVED** [review]

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (kyle4908) [reply docs/SRS-Volere/SRS.tex:780]: because we aren't going to be able to get all of them as soon as they are identified

Comment (RafeedIqbal) [reply docs/SRS-Volere/SRS.tex]: yep, added a timeframe

**Pull request merged**